tests/sparseml/pytorch/test_torch_to_onnx_exporter.py

```diff
@@ -24,13 +24,81 @@
 
 from sparseml.exporters.onnx_to_deepsparse import ONNXToDeepsparse
 from sparseml.pytorch.models.registry import ModelRegistry
+from sparseml.pytorch.optim import ScheduledModifierManager
 from sparseml.pytorch.sparsification.quantization import QuantizationModifier
 from sparseml.pytorch.torch_to_onnx_exporter import TorchToONNX
 from sparseml.pytorch.utils import ModuleExporter
 from sparsezoo.utils import validate_onnx
 from tests.sparseml.pytorch.helpers import ConvNet, LinearNet, MLPNet
 
 
+QUANT_RECIPE = """
+!QuantizationModifier
+    start_epoch: 0.0
+    scheme:
+        input_activations:
+            num_bits: 4
+            symmetric: False
```

Comment:
it's cool how simple it is to represent this in a recipe. could you also test where the activations are not quantized to the same num_bits as the weights? AFAIK we're targeting 8-bit activations at lowest precision